deploy/charts/harvester/Chart.yaml

```diff
@@ -49,5 +49,5 @@ dependencies:
     version: 0.1.0
     repository: https://kube-vip.io/helm-charts
   - name: harvester-load-balancer
-    version: 0.1.1
+    version: 0.1.2
```

Comment:
I saw these changes. Do we ought to run `helm dep update` under this directory for updating deps tarball?

Reply:
Added it. Thank you.